mod/wikirate/spec/set/all/contributions_spec.rb

```diff
@@ -39,7 +39,7 @@
       @direct_cc = @analysis.direct_contribution_count.to_i
       @subject.update_contribution_count
       @initial_count = @subject.contribution_count.to_i
-      article = @analysis.fetch :trait=>:wikirate_article,:new=>{}
+      article = @analysis.fetch :trait=>:overview,:new=>{}
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space missing after comma.<br>Surrounding space missing for operator `=>`.